src/esp/physics/objectWrappers/ManagedArticulatedObject.h

```diff
@@ -310,7 +310,7 @@ class ManagedArticulatedObject
   std::string getPhysObjInfoInternal(
       std::shared_ptr<esp::physics::ArticulatedObject>& sp) const override {
     // TODO fill out appropriate reporting values
-    std::string res = std::to_string(sp->getNumLinks()).append(1, ',');
+    std::string res{std::to_string(sp->getNumLinks())};
```

Comment:
Why even store in a temp string here? Just return to_string? It will be copied by the return value anyway if appropriate.